LogServer/src/main/java/org/isis/logserver/parser/XmlMessageParser.java

```diff
@@ -31,43 +31,24 @@
 import org.xml.sax.SAXException;
 
 /**
- * Parses messages that are received from the IOC in an XML format
- *
+ * Parses messages that are received from the IOC in an XML format.
  */
 public class XmlMessageParser
 {
-	public LogMessage parse(String text) 
-	{
-		LogMessage message = xmlToLogMessage(text);
-		
-		// message null if parse failed
-		if(message == null) {
-			return null;
-		}
-		
-		if(message.getContents() == null) {
-			message.setContents(text);
-		}
 
-		message.setRawMessage(text);
-		
-		return message;
-	}
-	
-	public LogMessage xmlToLogMessage(String xml)
+	public void parse(LogMessage message)
```

Comment:
This has side effects on the passed in message. I don't think we should do that.